File: practice/find_repeated_number.go
package practice

func ArrayCountValues(s []interface{}) map[interface{}]uint {
	r := make(map[interface{}]uint)
	for _, v := range s {
		if c, ok := r[v]; ok {
			r[v] = c + 1
		} else {
			r[v] = 1
		}
	}

	return r
}

func findRepeatedNumber(numbers []interface{}) []interface{} {
	var sameNumbers []interface{}
	checked_numbers := ArrayCountValues(numbers)
	for number, v := range checked_numbers {
		if v > 1 {
			sameNumbers = append(sameNumbers, number)
		}
	}
	return sameNumbers

}

File: practice/find_repeated_number.php
<?php
function findRepeatedNumber($numbers) {
    $sameNumbers = [];
    $checked_numbers = array_count_values($numbers);
    foreach ($checked_numbers as $number => $count) {
        if ($count > 1) {
            array_push($sameNumbers, $number);
        }
    }
    return $sameNumbers;

}

print_r(findRepeatedNumber([3,4,2,8,1,2,8]));

?>